### ISSUE TITLE
100% CPU when oc_id is running

### ISSUE THREAD

**Issue by xmik** [3ofcoins/docker-chef-server#1]
Hi, thanks for dockerizing Chef Server 12. I've be using it for a couple of months.
### Problem

From the beginning I was experiencing 100% CPU usage. I noticed a process which was being killed by sth and started again and again and again:

```
opscode    842 90.0  0.7  55548 16092 ?        Rsl  16:51   0:00 /opt/opscode/embedded/bin/ruby bin/rails server -p 9090 -b 127.0
```

I've read somewhere that this process is invoked by `oc_id` Chef component. `oc_id` is "OAuth2 provider for extensions like Analytics or Supermarket" as written on https://github.com/chef/chef-server . When I stopped that component with `chef-server-ctl stop oc_id`, there is no more the above process starting in a loop and CPU usage is lowered greatly.
### Hacky solution

In order to make those changes preserve chef-server container reboots, I replaced [this method](https://github.com/3ofcoins/docker-chef-server/blob/master/init.rb#L17) with:

``` ruby
def run!(*args, &block)
  log "Starting: #{args}" if ENV['DEBUG']
  pid = Process.spawn(*args)
  log "Started #{pid}: #{args.join ' '}"
  log "Stopping oc_id"
  `chef-server-ctl stop oc_id`
  $processes[pid] = block || ->{ log "#{args.join ' '}: #{$?}" }
  pid
end
```

This is obviously not pretty, but works.

I think it would be best to parametrize it: whether or not to stop (or start) oc_id at chef-server boot up. I don't use neither Analytics nor Supermarket nor Management Console, so I don't need oc_id, but I think others might need it. And of course it would be best to find a better implementation than this hacky solution. What do you think?

**Comment by xmik** [comment]
It is said to be resolved in Chef Server 12.6.0: https://github.com/chef/chef-server/issues/403 but I haven't tested it yet.

**Comment by mpasternacki** [comment]
Sorry about lack of responsiveness. I assume this issue is no longer relevant. Please reopen if it persists (but it seems like it's a bug in Chef Server itself rather than this image).
